.github/workflows/continuous-integration-linux-32bit.yml

```diff
@@ -4,15 +4,12 @@ on:
   push:
     branches:
       - develop
-  pull_request:
-    paths-ignore:
-    - '**/*.md'
-    types: [ opened, reopened, synchronize ]
+  workflow_call:
 
 permissions: read-all
 
 concurrency:
-  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-cilnx32
   cancel-in-progress: ${{github.event_name == 'pull_request'}}
```

Comment:
Is it usefult to set concurrency restrictins anywhere else than in the top level file?